### ISSUE TITLE
Instead of reading the color names, NVDA announces the RGB values of the colors

### ISSUE THREAD

**Issue by nunespe** [nvaccess/nvda#7943]
Excel 2010
Instead of reading the names of the cores, NVDA announces the RGB values of the cores in the cell color filter option.

Core names are not advertised in the Filter By option. NVDA announces RGB values.

Steps to Reproduce:

Create a title in cell A1, for example: "Tittle";
Place some fill color in cell A2 through the path: Home / Fill Color;
Enable the filter in cell A1 with Ctrl + Shift + L;
Use ALT + DOWN ARROW to open the context menu of the filter;
Scroll to a Color Filter option and open the submenu.

In this function NVDA does not read the name of the colors, only the RGB values.

below the link of an inssue solved that maybe can help in the resolution of this:

https://github.com/nvaccess/nvda/issues/3045

**Comment by Adriani90** [comment]
This is also reproducible in Excel 2016 and 365 with NVDA 2019.1.1. cc: @leonardder

**Comment by Adriani90** [comment]
This is still reproducible with NVDA 2024.4.1 in MS Excel 365. cc: @msftedad could you please report this to the development team?

**Comment by msftedad** [comment]
@Adriani90, We have forwarded this issue to development team. We will provide you with the tracking id once we receive update from them. Thanks

**Comment by msftedad** [comment]
@Adriani90, Issue shared with dependent team, for reference, team have created a ticket "issue [Accessibility 234563](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdev.azure.com%2FCSS-IRT%2FCSS%2520IRT%2F_workitems%2Fedit%2F234563&data=05%7C02%7Cv-ymounika%40microsoft.com%7C254f6f3c6f0040ee981108dd18844a63%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638693678665131774%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=liR47GuMDbC87UCsf4A99y5A1%2BpB7NZgO4qm2Fpt0tc%3D&reserved=0)" tracking purposes, you won’t have access to this link, but this can be used as a reference number for this request.
